src/log-io.js

```diff
@@ -92,8 +92,8 @@ class LogIO {
    * @param {function(hash, entry, parent, depth)} [onProgressCallback]
    * @returns {Promise<Log>}
    */
-  static async fromEntry (immutabledb, sourceEntries, length = -1, exclude, onProgressCallback) {
-    if (!isDefined(immutabledb)) throw LogError.ImmutableDBNotDefinedError()
+  static async fromEntry (ipfsf, sourceEntries, length = -1, exclude, onProgressCallback) {
```

Comment:
Should these be ```ipfs```? I think that's more consistent

Reply:
yup definitely